api/src/identity-access-management/application/user/user.admin.route.js

```diff
@@ -70,4 +72,52 @@ export const userAdminRoutes = [
       tags: ['api', 'admin', 'user'],
     },
   },
+  {
+    method: 'PATCH',
+    path: '/api/admin/users/{id}',
+    config: {
+      pre: [
+        {
+          method: (request, h) =>
+            securityPreHandlers.hasAtLeastOneAccessOf([
+              securityPreHandlers.checkAdminMemberHasRoleSuperAdmin,
+              securityPreHandlers.checkAdminMemberHasRoleSupport,
+            ])(request, h),
+        },
+      ],
+      plugins: {
+        'hapi-swagger': {
+          payloadType: 'form',
+        },
+      },
+      validate: {
+        params: Joi.object({
+          id: identifiersType.userId,
+        }),
+        payload: Joi.object({
+          data: {
+            attributes: {
+              'first-name': Joi.string().empty(Joi.string().regex(/^\s*$/)).required(),
+              'last-name': Joi.string().empty(Joi.string().regex(/^\s*$/)).required(),
+              email: Joi.string().email().allow(null).optional(),
+              username: Joi.string().allow(null).optional(),
+              lang: Joi.string().valid(...AVAILABLE_LANGUAGES),
+              locale: Joi.string()
+                .allow(null)
+                .optional()
+                .valid(...SUPPORTED_LOCALES),
+            },
+          },
+        }),
+        options: {
+          allowUnknown: true,
+        },
+      },
+      handler: (request, h) => userAdminController.updateUserDetailsByAdmin(request, h),
+      notes: [
+        "- Permet à un administrateur de mettre à jour certains attributs d'un utilisateur identifié par son identifiant",
+      ],
+      tags: ['api', 'admin', 'user'],
```

Comment:
Il serait bon de faire figurer le bounded context dans les tags de la route, comme dans https://github.com/1024pix/pix/blob/dev/api/src/identity-access-management/application/anonymization/anonymization.admin.route.js#L34

```suggestion
      tags: ['api', 'admin', 'identity-access-management'],
```